syft/serde/torch/serde.py

```diff
@@ -29,6 +29,16 @@
 TORCH_STR_DTYPE = {name: cls for cls, name in TORCH_DTYPE_STR.items()}
 
 
+TORCH_MFORMAT_ID = {
+    torch.channels_last: 1,
+    torch.contiguous_format: 2,
+    torch.contiguous_format: 3,
```

Comment:
There is two identcal keys here

Reply:
Yep. Tested again with
```
>>> for attr in dir(torch):
...     a = getattr(torch, attr)
...     if isinstance(a, torch.memory_format):
...             print(a)
...```
And the result is:
``` 
torch.channels_last
torch.contiguous_format
torch.preserve_format
```